sklearn/cross_decomposition/_pls.py

```diff
@@ -482,6 +482,20 @@ class PLSRegression(_PLS):
 
     Attributes
     ----------
+    x_mean_ : ndarray of shape (n_features,)
```

Comment:
I think we should add these at the end of the attributes section, since they're the least relevant attributes